### PR TITLE
Remove bintray's latest artifact generation

### PR DESCRIPTION
**Pull request by totahuanocotl**
Bintray does not allow to modify artifacts after 355 of publishing, so
there is no point to maintain a latest artifact that will go stale.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,11 +43,11 @@ binaries main use are the client commands.
 
 Osprey's executable binaries can be downloaded from our [Bintray repository](https://dl.bintray.com/sky-uk/oss-generic/osprey).
 
-To install a specific version replace `latest` with the release version.
+To install a specific version replace `<version>` with the release version.
 
 **Linux and Mac OS**
 ```
-  curl -L https://dl.bintray.com/sky-uk/oss-generic/osprey/latest/osprey-latest_linux_amd64.tar.gz -o osprey.tar.gz
+  curl -L https://dl.bintray.com/sky-uk/oss-generic/osprey/<version>/osprey-<version>_linux_amd64.tar.gz -o osprey.tar.gz
```
Comment (howardburgess) [inline README.md]:
```suggestion
  curl -fsSL https://dl.bintray.com/sky-uk/oss-generic/osprey/<version>/osprey-<version>_linux_amd64.tar.gz | tar -zxvC $HOME/.local/bin
```

Seem it is downloaded already with the correct permissions:
```bash
$ curl -fsSL https://dl.bintray.com/sky-uk/oss-generic/osprey/v2.1.0/osprey-v2.1.0_linux_amd64.tar.gz | tar -zxvC $HOME/.local/bin
x osprey
howard@narwhal ~/go/src/github.com/sky-uk/core-infrastructure/docker-alertmanager (increase-

$ ls -l ~/.local/bin/osprey
-rwxr-xr-x 1 howard staff 14700512 Nov  5 17:09 /Users/howard/.local/bin/osprey
```

Should we also emphasise that `$HOME/.local/bin` should be in their `$PATH`? Especially on macOS, that directory isn't added by default.

File: README.md [hunk 2]
```diff
@@ -43,11 +43,11 @@ binaries main use are the client commands.
 
 Osprey's executable binaries can be downloaded from our [Bintray repository](https://dl.bintray.com/sky-uk/oss-generic/osprey).
 
-To install a specific version replace `latest` with the release version.
+To install a specific version replace `<version>` with the release version.
```
Comment (howardburgess) [inline README.md]:
```suggestion
To install a specific version replace `<version>` with the release version (e.g. `v2.1.0`).
```

Not sure about including a version number that will go out of date, but I can imagine a common mistake would be to omit the `v`, and hence cause support questions. What do you think?

Comment (howardburgess) [reply README.md]: I suppose we could say `v9.9.9` as an obviously-wrong version but which illustrates the leading-`v` pattern.

File: README.md (hunk 1 above)
Comment (howardburgess) [reply README.md]: Related to comment below about the leading `v` - we could illustrate that here, with something like: `curl -fsSL https://dl.bintray.com/sky-uk/oss-generic/osprey/v<version>/`...

File: README.md [hunk 3]
```diff
@@ -43,21 +43,25 @@ binaries main use are the client commands.
 
 Osprey's executable binaries can be downloaded from our [Bintray repository](https://dl.bintray.com/sky-uk/oss-generic/osprey).
 
-To install a specific version replace `latest` with the release version.
+To install a specific version replace `<version>` with the release version
+(e.g `v9.9.0`; mind the `v` prefix).
```
Comment (totahuanocotl) [inline README.md]: will this do?

Comment (howardburgess) [reply README.md]: Lokoks fine, yep 👍

File: README.md (hunk 1 above)
Comment (howardburgess) [reply README.md]: I made the suggestion on one line, but it can actually replace lines 50-53 - doing the `curl` and un-`tar`ing in one line

**Pull request merged**